content/patterns/radio/examples/radio.html

```diff
@@ -207,13 +207,12 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>div</code></td>
               <td>
                 <ul>
-                  <li>Makes the radio button focusable and includes it in the page <kbd>Tab</kbd> sequence.</li>
-                  <li>Set on only one radio in the radio group.</li>
-                  <li>On page load, is set on the first radio button in the radio group.</li>
-                  <li>Moves with focus inside the radio group so the most recently focused radio button is included in the page <kbd>Tab</kbd> sequence.</li>
+                  <li>ラジオボタンをフォーカス可能にし、ページの <kbd>Tab</kbd> 順序に含めます。</li>
+                  <li>ラジオグループ内の 1 つのラジオにのみ設定されます。</li>
+                  <li>ページロード時に、ラジオグループの最初のラジオボタンに設定されます。</li>
+                  <li>ラジオグループ内のフォーカスと共に移動し、最近フォーカスされたラジオボタンがページの <kbd>Tab</kbd> 順序に含まれるようにします。</li>
```

Comment:
most recentlyをどう訳すか……
「最近」だと期間を表しているように感じられるので、「最後に」はどうでしょう？

あと用語
```suggestion
                  <li>ラジオグループ内のフォーカスと共に移動し、最後にフォーカスされたラジオボタンがページの <kbd>Tab</kbd> シーケンスに含まれるようにします。</li>
```